src/core/a-entity.js

```diff
@@ -127,6 +129,28 @@ var proto = Object.create(ANode.prototype, {
     }
   },
 
+  getObject3D: {
+    value: function (type, Constructor) {
+      var object3D = this.objects3D[type];
+      if (!object3D && Constructor) {
+        object3D = this.objects3D[type] = new Constructor();
```

Comment:
can we have some default types where we know what the constructor should be? like if `type` is `mesh`, do we have to specify the constructor since we know to use `THREE.Mesh`?